src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/Application.js

```diff
@@ -69,8 +69,8 @@ class Application extends React.Component<Props>{
 
     navigationPinnedDisposer: () => void;
 
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
```

Comment:
:scream: Did that also make any actual problems?

Reply:
Didn't make any problems did try many things today and found out that this was missing here 😄